raiden/tasks.py

```diff
@@ -91,7 +91,7 @@ def poll_for_new_block(self):
             self.last_block_number = current_block
             remove = list()
             for callback in self.callbacks:
-                result = callback(current_block)
+                result = callback(current_block, self.chain.network_id())
```

Comment:
humm, this is an additional rpc call for each new block and callback, since we don't expect this value to change, cant you move it to the task constructor?

Reply:
It can change if the underlying client changes the network. Also it's a cached value every 10 seconds. 

Moving it out of the loop here and I will increase the TTL value in the blockchain_service which should reduce the number of RPC calls considerably while still allowing us to handle cases where the underlying chain_id changes without changing the alarm task.

I do realize that right now we would not handle such a scenario very well anyway but I think it's cleaner and will make it easier in the future.